ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/AwsConnectorExecutor.java

```diff
@@ -101,6 +101,9 @@ public void invokeRemoteModel(MLInput mlInput, Map<String, String> parameters, S
                 throw new OpenSearchStatusException("No response from model", RestStatus.BAD_REQUEST);
             }
             String modelResponse = responseBuilder.toString();
+            if (statusCode < 200 || statusCode >= 300) {
```

Comment:
is there a reason why are not checking 201 or other status code?

Reply:
That depends on the remote model service to determine return 200 or other 2xx code.

Check HTTP RFC 
> 2xx (Successful): The request was successfully received,
      understood, and accepted

Reply:
So we are treating all 2XX as successful response.

Reply:
Yes, because that's what HTTP RFC defines